user_docs/global.t2tconf

```diff
@@ -1,22 +1,22 @@
-%!Target: html
+%!Target: xhtml

```

Comment:
Why do we need to change the format?

Reply:
This is not necessary at this point, but I had been working on parsing the user guide as XML instead of treating it as plain text.